src/commands/new.rs

```diff
@@ -1,21 +1,37 @@
 use std::path::PathBuf;
-
-use clap::{Parser};
+use clap::{Parser, ArgGroup};
 use itertools::Itertools;
-
 use crate::hub_api;
 
 #[derive(Parser, Debug)]
 #[clap(about = "Create an application from a template on the Hub")]
+#[clap(group(
+    ArgGroup::new("operation")
+        .required(true)
+        .args(&["list", "name"])
+))]
 pub struct NewCommand {
     #[clap(short = 't')]
     terms: Vec<String>,
 
-    name: String,
+    #[clap(short, long)]
+    list: bool,
```

Comment:
I'm not sure this is needed on `new`.  The `search` command already exists for this.